src/components/UI/Input/InputTypeTwo.jsx

```diff
@@ -0,0 +1,40 @@
+import "./InputTypeTwo.css";
+export const InputTypeTwo = (props) => {
+	const {
+		wrapperClassName,
+		htmlFor,
+		labelClassName,
+		labelText,
+		type,
+		className,
+		placeholder,
+		name,
+		id,
+		required,
```

Comment:
You can do this for you ternary operator

InputComponent =requried=>{

let required = required ? "required": " ";
}